### PR TITLE
Tinglis/add missed activities

### PR DESCRIPTION
**Pull request by tinglisD2L**

### CODE REVIEW

File: d2l-sequence-viewer.html [hunk 1]
```diff
@@ -249,7 +267,9 @@
 				this._toastListener = null;
 			}
 			_closeSlidebar() {
-				this.$.sidebar.classList.add('offscreen');
+				if ( document.hasFocus() ) {
```
Comment (lawong-d2l) [inline d2l-sequence-viewer.html]: Is this left over from another branch?

File: d2l-sequence-viewer.html [hunk 2]
```diff
@@ -132,17 +132,28 @@
...
+				</d2l-sequences-content-eol-main>
+			</template>
+			<template is="dom-if" if="[[!isEndOfLesson]]">
+				<d2l-sequences-content-router
+					class="viewer"
+					href="[[href]]"
+					token="[[token]]"
+				></d2l-sequences-content-router>
+			</template>
+			<d2l-alert-toast
```
Comment (lawong-d2l) [inline d2l-sequence-viewer.html]: Was this d2l-alert-toast change supposed to be in here?

File: d2l-sequence-viewer.html (hunk 1 above)
Comment (tinglisD2L) [reply d2l-sequence-viewer.html]: oh that was in the WIP don't merge branch you sent me. I thought it was something you wanted to include. I'll remove it.

File: d2l-sequence-viewer.html (hunk 2 above)
Comment (tinglisD2L) [reply d2l-sequence-viewer.html]: i'll check

**Pull request merged**